### PR TITLE
Update crf sample

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Added a sample app to this repo that does not connect to bridge.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "BridgeApp"]
 	path = BridgeApp
 	url = https://github.com/Sage-Bionetworks/BridgeApp-Apple-SDK.git
+[submodule "SageResearch"]
```
Comment (Erin-Mounts) [inline .gitmodules]: will this end up with two copies of SageResearch in this tree (one under BridgeApp)?

**Review by Erin-Mounts — APPROVED** [review]
One question -- otherwise looks good as far as I can tell

File: .gitmodules (hunk 1 above)
Comment (Erin-Mounts) [reply .gitmodules]: never mind, I think I figured out why you're doing that.

**Comment by Erin-Mounts** [comment]
I think you need to update the xcode version in .travis.yml

File: .gitmodules (hunk 1 above)
Comment (syoung-smallwisdom) [reply .gitmodules]: I didn't remove our validation app b/c I want to be able to use it to get data to Megha, but this does not reference that version.

**Pull request merged**